src/ompl/geometric/planners/bitstar/src/BITstar.cpp

```diff
@@ -1505,28 +1520,21 @@ namespace ompl
         bool BITstar::isCostWorseThanOrEquivalentTo(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
             //If a is not better than b, than a is worse than, or equal to, b
-            return !this->isCostBetterThan(a,b);
+            return !opt_->isCostBetterThan(a,b);
         }
 
 
 
         bool BITstar::isFinite(const ompl::base::Cost& cost) const
         {
-            return this->isCostBetterThan(cost, opt_->infiniteCost());
+            return opt_->isCostBetterThan(cost, opt_->infiniteCost());
         }
 
 
 
         ompl::base::Cost BITstar::betterCost(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
-            if (this->isCostBetterThan(b,a))
-            {
-                return b;
-            }
-            else
-            {
-                return a;
-            }
+            return opt_->isCostBetterThan(a, b) ? a : b;
```

Comment:
Ahah, I understand that you are someone very rigorous and it's a good thing. Sorry, I just did it automatically :smile: